### PR TITLE
docs: fixing onboarding documentation omissions & adding a 'troubleshooting' section

### PR DESCRIPTION
**Pull request by GaelFerrand**
# Summary

In CONTRIBUTING.md
- Fixing the docker-compose line to include the mongodb container
- Adding a troubleshooting section with:
  - How to solve the default$default issue
  - How to solve the elastic search index export issue

Check the updated file [here](https://github.com/MTES-MCT/trackdechets/blob/f0d4e2876ccd564c1bab7b04501c5003bda87e93/CONTRIBUTING.md)

Also fixed the `.integration-tests-env` file to use single quotes for the postgres DB path

# Checklist

- [x] Mettre à jour la documentation

(pas de ticket favro)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,6 +20,9 @@
     - [Créer un tampon de signature pour la génération PDF](#créer-un-tampon-de-signature-pour-la-génération-pdf)
```
Comment (GaelFerrand) [inline CONTRIBUTING.md:20]: The more human-readeabl, interpreted version [here](https://github.com/MTES-MCT/trackdechets/blob/f0d4e2876ccd564c1bab7b04501c5003bda87e93/CONTRIBUTING.md#d%C3%A9pannage)

**Review by providenz — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -322,3 +325,52 @@ Pour s'y retrouver plus facilement, suivre la convention de nommage en place et
...
+docker exec -it $(docker ps -qf "name=td-api") bash
+
+export FORCE_LOGGER_CONSOLE=true
+
+npm run reindex-all-bsds-bulk:dev -- -f
+```
+
+Si le problème remonté est un "Segmentation fault", il est probable que la mémoire allouée au container soit insuffisante. Vous pouvez contourner le problème en limitant la taille des batches (dans votre .env):
+```
+BULK_INDEX_BATCH_SIZE=100
```
Comment (elishowk) [inline CONTRIBUTING.md:358]: je mettrai bien aussi par défaut 100 ici https://github.com/MTES-MCT/trackdechets/blob/e4187500a5cb5d7acc3743eebbd959ba2d1fa8e1/back/src/bsds/indexation/bulkIndexBsds.ts#L540-L541

**Review by elishowk — CHANGES_REQUESTED** [review]
Merci, une suggestion proposée en commentaire sur https://github.com/MTES-MCT/trackdechets/blob/e4187500a5cb5d7acc3743eebbd959ba2d1fa8e1/back/src/bsds/indexation/bulkIndexBsds.ts#L540-L541

File: CONTRIBUTING.md (hunk 2 above)
Comment (GaelFerrand) [reply CONTRIBUTING.md:358]: Bonne idée, done!

**Pull request merged**